src/log.rs

```diff
@@ -0,0 +1,28 @@
+static mut ENABLED: bool = false;
```

Comment:
Adding a mutex here will remove unsafe code when accessing this variable

Reply:
You're right, thanks! :D